src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: LAMPy stack monitoring integration
+tags:
+    - New Relic integrations
+    - LAMPy
+    - Linux
+    - Apache
+    - Python
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
```

Comment:
metaDescription: Use the below New Relic's agents to get a dashboard with metrics for your LAMPy application.
1. Infrastructure agent
2. Apache agent
3. MySQL agent
4. Python agent